app/views/patients/update.js.erb

```diff
@@ -36,3 +36,14 @@ $('#change_log').html(
   // clear out fulfillment form
   $('#fulfillment').empty();
 <% end %>
+
+// flash success
+if ($( "#flash" ).length == 0) {
+  $('main > .container').prepend("<div id='flash'></div>")
+}
+
+$('#flash').removeClass().hide().fadeIn('slow')
+$('#flash').addClass("alert alert-success")
+$('#flash').html(
+  '<button type="button" class="close" data-dismiss="alert" aria-hidden="true">×</button><div id="flash_notice"> Patient data saved!</div>'
```

Comment:
to be clear, here's what the relevant part of `update.js.erb` looks like in here:

```js
// flash success
// if ($( "#flash" ).length == 0) {
  // $('main > .container').prepend("<div id='flash'></div>")
// }

// $('#flash').addClass("alert alert-success")
$('#flash').html('<%= escape_javascript(render partial: "layouts/messages") %>');
$('#flash').removeClass().hide().fadeIn('slow')
```